universal-application-tool-0.0.1/app/controllers/admin/ProgramAdminManagementController.java

```diff
@@ -86,8 +87,24 @@ public Result update(Http.Request request, long programId) {
           roleService.makeProgramAdmins(
               programId, ImmutableSet.copyOf(manageAdminForm.getAdminEmails()));
       Result result = redirect(routes.AdminProgramController.index());
-      if (maybeError.isPresent()) {
-        return result.flashing("error", maybeError.get().message());
+
+      if (maybeError.isPresent()){   
```

Comment:
nit: change to `if (!maybeError.isPresent())` with an early return so that the bulk of the new code isn't at an additional level of indention. also avoids nested `if`s (even though cyclomatic complexity is identical)

Reply:
Done.